### DISCUSSION TITLE
Max/Min Values

### DISCUSSION THREAD

**Discussion by christianaz** [FrandsenGroup/beams#94] [Q&A] [answered]
Hello! I had a quick question about setting upper/lower bounds when fitting functions. I tried to plug in numbers into the max and min columns when doing a fit, but it wouldn't do the fit at all. Am I just supposed to put numbers in or is there something else I need to do? The "Your fit is running" window pops up, then closes, but nothing is graphed. (Note: it does work every once in a while, but not consistently)

**Comment by aPeter1** [comment, marked as answer]
I added some input checking and dialog windows that will pop up to notify you if the input is invalid and how it is invalid in the case of bounds. Also, when a fit failed (because it was on a separate process) it didn't crash the program but I also wasn't reporting it the user so it will do that now as well. Let me know if you still have an issue, hopefully this will help pin it down if there is one. (I opened an issue #95 for this, feel free to reopen it if you find something). Thanks!

**Comment by christianaz** [comment]
The changes were very helpful, thank you so much!
